assets/wire/reducers.js

```diff
@@ -111,16 +112,21 @@ function updateItemActions(state, items, action) {
 
 function _wireReducer(state, action) {
     switch (action.type) {
-    case TOGGLE_SERVICE: {
-        const activeService = {};
-        if (action.service) {
-            activeService[action.service.code] = true;
+
+    case TOGGLE_NAVIGATION: {
```

Comment:
imo it would be better if we only deal with navigation on client side and resolve products on server. it's an implementation detail we don't have to expose

Reply:
Done that